tests/plugins/test_health_checks.py

```diff
@@ -25,6 +26,7 @@ def plugin_manager_with_doctor_command(plugin_manager):
     Registers the `conda doctor` subcommand
     """
     plugin_manager.load_plugins(doctor)
+    plugin_manager.load_plugins(*reporter_backends_plugins)
```

Comment:
That seems not ideal to handle for every test module, let's add a fixture to install must-have plugins in the test suite?